package.json

```diff
@@ -45,7 +45,10 @@
     "test:position-change": "anchor test --skip-build tests-staker/position-change.spec.ts",
     "build:invariant": "cd sdk && npm run build",
     "build:staker": "cd staker-sdk && npm run build",
-    "build:all": "npm run build:invariant && npm run build:staker"
+    "build:all": "npm run build:invariant && npm run build:staker",
+    "program:jupiter": "anchor build -- --features \"jupiter\"",
```

Comment:
all `program:*` scripts failed

Reply:
so, should I delete them?

Reply:
nope, I mean they should end successfully, not fail :)